pp/Actions/actions.pd

```diff
@@ -36,6 +37,15 @@ use version;
 my $pdl_version = version->parse( $PDL::Version::VERSION );
 my $propagate_badflag = $pdl_version >= '2.008' ? 'propagate_badflag' : 'propogate_badflag';
 
+our @_REAL_TYPES =
```

Comment:
Please use this idiom: `map $_->ppsym, grep $_->real, PDL::Types::types()`

Reply:
That would be the simplest solution. I just wanted some backward compatibility, but if people are installing PDL::NDBin, they probably will also be upgrading PDL.

@ebaudrez, do you want to bump the PDL version to 2.034 which introduces `$type->real`?

Reply:
I would venture that the simplest way is to add a commit that executes this, so he could either merge the whole thing, or just cherry-pick out everything but that.

Reply:
Done.